frontend/package.json

```diff
@@ -140,6 +142,7 @@
     "ts-action-operators": "^9.1.2",
     "typedjson": "^1.5.1",
     "urijs": "^1.19.11",
+    "uuid": "^8.3.2",
```

Comment:
What is the other one? This one is the actual dependency which was just added here because we now use it explicitly. It was already in the dependency tree prior to these changes. The other one is just typings and only a dev dependency.

Reply:
Ah okay, my bad. I just saw two additions in this file with `uuid` and assumed that they were two different packages. You can ignore this 👍